src/main/java/com/checkout/payments/ProcessingSettings.java

```diff
@@ -37,4 +43,32 @@ public final class ProcessingSettings {
     @SerializedName("receipt_id")
     private String receiptId;
 
+    @SerializedName("terminal_type")
+    private TerminalType terminalType;
+
+    @SerializedName("os_type")
+    private OsType osType;
+
+    @SerializedName("invoice_id")
+    private String invoiceId;
+
+    @SerializedName("brand_name")
+    private String brandName;
+
+    private String locale;
+
+    @SerializedName("shipping_preference")
+    private ShippingPreference shippingPreference;
+
+    @SerializedName("user_action")
+    private UserAction userAction;
+
+    @SerializedName("set_transaction_context")
+    private List<Map<String, String>> setTransactionContext;
+
+    @SerializedName("airline_data")
+    private List<AirlineData> airlineData;
+
+
+
```

Comment:
Just a visual comment. Maybe delete these lines and add a new line at the end of file.

Reply:
Woops done, thanks!